bin/cml/comment/create.js

```diff
@@ -16,6 +16,15 @@ exports.options = kebabcaseKeys({
     description:
       'Post to an existing PR/MR associated with the specified commit'
   },
+  watch: {
```

Comment:
Why the move?

`--pr` and `--commit-sha` represent comment targets, so arguably the flags should be listed next to each other.